tests/analysis/losses/test_losses.py

```diff
@@ -9,26 +9,32 @@
     AnalysisConfigData,
     AnalysisSectionLosses,
 )
-from ra2ce.analysis.analysis_config_data.enums.analysis_losses_enum import (
-    AnalysisLossesEnum,
-)
 from ra2ce.analysis.analysis_config_data.enums.trip_purposes import TripPurposeEnum
 from ra2ce.analysis.analysis_config_data.enums.weighing_enum import WeighingEnum
 from ra2ce.analysis.analysis_config_wrapper import AnalysisConfigWrapper
 from ra2ce.analysis.analysis_input_wrapper import AnalysisInputWrapper
-from ra2ce.analysis.losses.losses import Losses
+from ra2ce.analysis.losses.losses_base import LossesBase
+from ra2ce.analysis.losses.single_link_losses import SingleLinkLosses
 from ra2ce.network.network_config_data.enums.part_of_day_enum import PartOfDayEnum
 from ra2ce.network.network_config_wrapper import NetworkConfigWrapper
 from tests import test_data
 
 
 class TestLosses:
+    def test_initialize_base_class_raises(self):
+        # 1. Run test.
+        with pytest.raises(TypeError) as exc:
+            _losses = LossesBase(None, None)
+
+        # 2. Verify final expectations
+        assert str(exc.value).startswith("Can't instantiate abstract class LossesBase")
```

Comment:
Usually you want to prevent hardcoding this by doing a direct reference to the class itself. This way if there's a 'refactoring' it gets updated automatically.
```suggestion
        assert str(exc.value).startswith(f"Can't instantiate abstract class {LossesBase.__name__}")
```